doc/index.rst

```diff
@@ -74,4 +74,4 @@ posted in the `Issues
 .. |Python 37| image:: https://img.shields.io/badge/Python-3.7-brightgreen.svg?style=flat
    :target: http://python.org
 .. |Python 38| image:: https://img.shields.io/badge/Python-3.8-brightgreen.svg?style=flat
-   :target: http://python.org
\ No newline at end of file
+   :target: http://python.org
```

Comment:
Was this (and similar) doc changes for line length? It is generally better to keep cosmetic/unrelated changes separate for clarity of a feature patch.

Reply:
I removed cosmetic/unrelated changes from history